scraper/src/kolibri2zim/scraper.py

```diff
@@ -1047,10 +1221,11 @@ def add_custom_about_and_css(self):
         )
         with self.creator_lock:
             self.creator.add_item_for(
-                path="about",
+                path="files/about",
```

Comment:
`files/about` because all content is placed in `files` now
not front because it is just an about page containing copyright information and stuff like that, so I do not expect people to be interesting in getting this page in suggestion, random pages, ... ; but I agree this is debatable

Reply:
Can you list all prefixes in use with your code and what goes inside? It seems your PR comment is not complete (no mention of `files/` for instance).

Reply:
All prefix are listed in the PR, I just didn't reflected there the fact that `/static` has been renamed `/files` upon your request.

Reply:
I have difficulties not considering this content. Particularily in the case it's provided by user. It's relevance or quality is a distinct topic.

I also notice you removed the link to this page and did not create a ticket to add it back. Is that on purpose?

Reply:
OK, I've set is_front back to True
And no the about link was missing ... because it wasn't present in endless design + I forgot about it. 😄 It's back now.